### PR TITLE
Ex UI 1315 incorrect category ordering in case view file

### PR DESCRIPTION
**Pull request by OgunyemiO**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EXUI-1315

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts [hunk 1]
```diff
@@ -109,11 +109,12 @@ export class CaseFileViewFolderComponent implements OnInit, OnDestroy {
       newDocumentTreeNode.name = node.category_name;
       newDocumentTreeNode.type = DocumentTreeNodeType.FOLDER;
       newDocumentTreeNode.children = [...this.generateTreeData(node.sub_categories), ...this.getDocuments(node.documents)];
+      newDocumentTreeNode.category_order = node.category_order;
 
       return [
         ...tree,
         newDocumentTreeNode,
-      ];
+      ]//.sort((a,b) => a.category_order - b.category_order);
```
Comment (connorpgpmcelroy) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts:117]: Is this comment necessary?

**Review by connorpgpmcelroy — APPROVED** [review]
Approved with minor comment

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts (hunk 1 above)
Comment (OgunyemiO) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts]: @connorpgpmcelroy comment removed. Could you please reapprove?

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM

**Review by RiteshHMCTS — APPROVED** [review]

**Review by olusegz07 — DISMISSED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts [hunk 2]
```diff
@@ -83,7 +83,7 @@ describe('CaseFileViewFolderComponent', () => {
     fixture.detectChanges();
   }));
 
-  it('should create', async() => {
+  it('should create', async () => {
```
Comment (andywilkinshmcts) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts:86]: Incorrect whitespace change - is your IDE misconfigured?

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts [hunk 3]
```diff
@@ -74,7 +74,7 @@ describe('CaseFileViewFolderComponent', () => {
         { provide: AbstractAppConfig, useValue: mockAppConfig }
       ]
     })
-    .compileComponents();
+      .compileComponents();
```
Comment (andywilkinshmcts) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts:77]: Unnecessary white space change

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts [hunk 4]
```diff
@@ -295,7 +294,7 @@ describe('CaseFileViewFolderComponent', () => {
     });
   });
 
-  it('should filter documents verify UI', async() => {
+  it('should filter documents verify UI', async () => {
```
Comment (andywilkinshmcts) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts:297]: Incorrect whitespace change - please check your IDE settings

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts [hunk 5]
```diff
@@ -309,7 +308,7 @@ describe('CaseFileViewFolderComponent', () => {
     expect(documentTreeContainerEl.textContent).toContain('Beers encyclopedia');
   });
 
-  it('should filter documents no match verify UI', async() => {
+  it('should filter documents no match verify UI', async () => {
```
Comment (andywilkinshmcts) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts:311]: And again

**Review by andywilkinshmcts — CHANGES_REQUESTED** [review]
Your IDE seems to be configured to add a space after async and before the (). Please change this and update the PR so it doesn't have these unnecessary whitespace changes

**Review by olusegz07 — DISMISSED** [review]

**Review by olusegz07 — APPROVED** [review]

**Review by andywilkinshmcts — APPROVED** [review]

**Review by olusegz07 — APPROVED** [review]

**Pull request merged**